Cargo.toml

```diff
@@ -3,6 +3,12 @@ name = "microvmi"
 version = "0.1.0"
 authors = ["Mathieu Tarral <mathieu.tarral@protonmail.com>"]
 edition = "2018"
+license = "GPL-3.0-only"
+description = "A cross-platform, unified, low-level VMI API on top of hypervisors"
```

Comment:
@rageagainsthepc I'd like your opinion on the description.
how do you feel when reading it ?
can we do better ?
my english is not incredible.

Reply:
```suggestion
description = "A cross-platform, unified, low-level VMI API supporting multiple hypervisors"
```
Perhaps?

Reply:
Hm, maybe you can scratch that idea. The word "unified" already gives away that multiple hypervisor are supported. lgtm

Reply:
Unified means to me "abstraction layer", so I'll keep it for now